src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_MatrixTranform.cs

```diff
@@ -105,12 +86,10 @@ public async Task When_Rotate_And_TransformBounds()
 					SUT.TransformBounds(new Rect(1, 1, 5, 6))
 				);
 			});
-		}
 
 		[TestMethod]
-		public async Task When_RotateQuarter_And_TransformPoint()
-		{
-			await Dispatch(() =>
```

Comment:
For UI tests, I prefer to always await **in** the method, i't  much easier to track exceptions!